docs/README.md

```diff
@@ -0,0 +1,328 @@
+
+### Model danych
+
+Model danych jest dostępny w dokumentacji API.
+
+Wyróżniamy następujące elementy:
+
+* Dictionary - dane na temat słowników danych statystycznych
+* Case - dane na temat sprawy
+* Institution - informacje o instytucjach
+* Tag - dodatkowa konfiguracja prezentacji znaczników
+* Collection - kolekcje umożliwiające dokonanie wyboru prezentowanych spraw poprzez wskazanie kryterium ich wyszukiwania
+* Letter - dane na temat listów przypisanych do sprawy / nowych
+* File - dane na temat plików powiązanych z listem
+* Channel - słownik zawierający konfiguracje kanałów komunikacji
+* Note - notatka wprowadzona przez użytkownika
+* Event - dane na temat wydarzenia terminarza
+* User - dane na temat użytkowników systemu
+* Description - słownik opisów listów
+* AdministrativeUnit - jednostki podziału terytorialnego
+
+### Model uprawnień
+
+System wyróżnia następujące uprawnienia:
+
+* Dictionary.change
+* Dictionary.view
+* Dictionary.delete
+* Dictionary.create
+* Case.change
+* Case.view
+* Case.delete
+* Case.create
+* Institution.change
+* Institution.view
+* Institution.delete
+* Institution.create
+* Tag.change
+* Tag.view
+* Tag.delete
+* Tag.create
+* Collection.change_own
+* Collection.change_all
+* Collection.view
+* Collection.delete
+* Collection.create
+* Letter.change_own
+* Letter.change_all
+* Letter.view
+* Letter.delete
+* Letter.create
+* Channel.change
+* Channel.view
+* Channel.delete
+* Channel.create
+* Note.change_own
+* Note.change_all
+* Note.view
+* Note.delete
+* Note.create
+* Event.change_own
+* Event.change_all
+* Event.view
+* Event.delete
+* Event.create
+* User.change_own
+* User.change_all
+* User.view
+* User.delete
+* User.create
+* Description.change
+* Description.view
+* Description.delete
+* Description.create
+* AdministrativeUnit.change
+* AdministrativeUnit.view
+* AdministrativeUnit.delete
+* AdministrativeUnit.create
+
+Klasy uprawnień mają następujące znaczenie:
+
+* change - edycja wszystkich elementów danego typu
+* view - wyświetlanie wszystkich elementów danego typu
+* delete - usunięcie wszystkich elementów danego typu
+* change_own - edycja elementów danego typu utworzonych przez tego samego użytkownika
+* change_all - edycja elementów danego typu utworzonych przez dowolnego użytkownika
+
+W celu kontroli dostępu wykorzystywane jest:
+
+* uwierzytelnianie  poprzez GSuite - oznaczone w dokumentacji API jako ```sessionAuth```
+* uwierzytelnianie hasłem - wyłącznie w celach administracyjnych, tożsame z oznaczeniem w API jako ```sessionAuth```
+* uwierzytelnianie kodem JWT - oznaczone w dokumentacji API jako ```bearerAuth```.
+
+Dokumentacja API pomija uwierzytelnianie w ten sposób jako standardowy protokół.
+
+### Model powiadomień
+
+Model powiadomień zostanie zdefiniowany po zrealizowaniu podstawowych elementów edycji.
+
+## Panel Zarządzania
```

Comment:
>> W jaki sposób będziemy projektować UI? Czy macie w tym zakresie preferowane rozwiązania? Wskazywano na kompetencje w zakresie front-endu po Waszej stronie.
> 
> Optymalnie: UXowiec -> grafik -> developer, ze współpracą z Product Ownerem na każdym etapie. Dobrze byłoby, żeby dev także miał wgląd na bieżąco (w razie gdyby pojawiły się jakieś pomysły trudne do faktycznej realizacji).
> 
> Można pominąć grafika, korzystając z którejś z gotowych bibliotek komponentów, np. do Material Design (są takie i do Vue, i do Reacta). To wygląda dobrze, działa, jest dobrze przetestowane i funkcjonalne. Jeśli nie potrzeba wodotrysków, to w zupełności się sprawdzi. Wtedy byłoby: UXowiec mający świadomość, jakie komponenty UI są do dyspozycji -> dev.
> 
> Wersja minimum: dev, mający jakiekolwiek pojęcie o UX i projektowaniu UI, robi projekt w bliskiej współpracy z PO, a potem implementuje.
> 
> Rezultatem powinny być wireframe'y, projekt interakcji i projekt graficzny (ten można ew. pominąć, jeśli skorzystać z biblioteki komponentów).

Reply:
W zakresie grafika stanowczo zwrócę uwagę, że projekt nie ma bezpośrednio funkcji promocji Stowarzyszenia, a pełni funkcje back-office. Ważne jest, aby dostarczyć to sprawnie, aby w pierwszym etapie biuro mogło czerpać korzyśc z nowych możliwość, następnie będą mogli korzystać członkowie Stowarzyszenia i najbliższe otoczenie. Wówczas na bazie powstałego systemu będzie można myśleć o opracowaniu formę prezentacji zintegrowaną z siecobywatelska.pl lub inny optymalny sposób. Dotąd bazowaliśmy w zakresie grafiki na django-admin, więc poprzeczka nie jest wysoko. Spełniało to swoją funkcje, ale nie było możliwe do dostatecznego dalszego rozwoju.

Wymieniono szereg osób i funkcji, a dotychczas w Stowarzyszeniu systemy były nierzadko rozwijane praktycznie przez pojedyncze osoby full-stack sporadycznie wspierane w razie potrzeby przez osoby wyspecjalizowane w danym obszarze. Dostrzegam, że jest to inny styl pracy niż w SII na co dzień.